common/djangoapps/student/urls.py

```diff
@@ -6,50 +6,50 @@
 from django.conf.urls import url
 from django.contrib.auth.views import password_reset_complete
 
-import student.views
+import views
```

Comment:
Would `from . import views` be better?  Python 3 no longer supports implicit relative imports.